HISTORY.rst

```diff
@@ -3,6 +3,88 @@
 History
 =======
 
+[PENDING]
```

Comment:
Since this PR contains the version bump, you can go ahead and put the new version here.  It's ok to guess at the date of the actual release, it doesn't matter much if it ends up being a day or two off.